src/rocm_docs/theme.py

```diff
@@ -21,14 +21,17 @@
 
 
 def _get_version_from_url(url: str) -> str:
+    headers = {
+        "User-Agent": "alexxu-amd"
```

Comment:
based on the doc https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
maybe this should be rocm-docs-core as the product name?

Reply:
I was looking at this doc from Github: https://docs.github.com/en/rest/using-the-rest-api/getting-started-with-the-rest-api?apiVersion=2022-11-28#user-agent, Github recommends using github username or the name of the application as User-Agent.

I wasn't sure if rocm-docs-core counts as an application, but if it is, it makes more sense to use it rather than my personal github name.

Reply:
let's keep username for now then